src/style.css

```
@@ -0,0 +1,71 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  text-decoration: none;
+  list-style: none;
+}
+
+.gadget {
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  margin-left: 21px;
+  line-height: 60px;
+  color: #000;
+  display: flex;
+}
+
+.box {
+  display: flex;
+  flex-direction: row;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.is-active {
+  margin-left: 20px;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+}
+
+.gadget:hover {
```

you can write like this
```suggestion
.gadget:hover, .is-active {
```